Extension/src/LanguageServer/client.ts

```diff
@@ -2724,15 +2730,20 @@ export class DefaultClient implements Client {
 
         showConfigStatus = showConfigStatus || (configurationNotSet &&
             !!compilerDefaults && !compilerDefaults.trustedCompilerFound && trustedCompilerPaths && (trustedCompilerPaths.length !== 1 || trustedCompilerPaths[0] !== ""));
+        if (showConfigStatus) {
+            this.showConfigureIntelliSenseButton = true;
+        } else {
+            this.showConfigureIntelliSenseButton = false;
+        }
+        const configType: ConfigurationType =
```

Comment:
Should AutoConfigProvider be handled here?

Reply:
yes